docs/src/main/paradox/scala/http/routing-dsl/index.md

```diff
@@ -45,11 +49,16 @@ in the Akka documentation.
 To help start a server Akka HTTP provides an experimental helper class called `HttpApp`.
 This is the same example as before rewritten using `HttpApp`:
 
-@@snip [HttpAppExampleSpec.scala]($test$/scala/docs/http/scaladsl/HttpAppExampleSpec.scala) { #minimal-routing-example }
+Scala
+:  @@snip [HttpAppExampleSpec.scala]($test$/scala/docs/http/scaladsl/HttpAppExampleSpec.scala) { #minimal-routing-example }
+
+Java
+:  @@snip [HttpAppExampleTest.java]($test$/java/docs/http/javadsl/server/HttpAppExampleTest.java) { #minimal-routing-example }
 
 See @ref[HttpApp Bootstrap](HttpApp.md) for more details about setting up a server using this approach.
 
-<a id="long-example"></a>
+@@@ div { .group-scala }
```

Comment:
Pity to have this only for Scala, but this can come in another PR

Reply:
Added #1639